test/inductor/test_cpu_repro.py

```diff
@@ -3750,7 +3750,7 @@ def fn(arg0_1):
         _, code = run_and_get_cpp_code(opt_fn, x)
         FileCheck().check_count(
             "return at::vec::VectorizedN<int64_t,2>::loadu(tmpbuf.data(),",
-            2,
+            4,
```

Comment:
Out of scope for this PR, but can you open an issue for this. The generated code is really dumb:
```cpp
                auto tmp27 =
                [&]
                {
                    __at_align__ std::array<int64_t, 8> tmpbuf;
                    tmp26.store(tmpbuf.data());
                    return tmpbuf;
                }
                ()
                ;
                auto tmp28 =
                [&]
                {
                    __at_align__ std::array<int64_t, 8> tmpbuf;
                    #pragma GCC unroll 8
                    for (long x0_inner = 0; x0_inner < 8; x0_inner++)
                    {
                        tmpbuf[x0_inner] = static_cast<long>(tmp27[x0_inner]);
                    }
                    return at::vec::VectorizedN<int64_t,2>::loadu(tmpbuf.data(), 8);
                }
                ()
                ;
```

`tmp27` is `tmp26` converted from `VectorizedN<int64_t, 2>` to a stack array. Then `tmp28` is it converted straight back to `VectorizedN<int64_t, 2>` which seems silly.

cc @jgong5